erroranalysis/requirements-dev.txt

```diff
@@ -4,4 +4,5 @@ pytest-mock==3.6.1
 
 requirements-parser==0.2.0
 rai_test_utils[object_detection]
+scikit-learn<=1.3.2
```

Comment:
this can be removed now as I've made the tests work with latest scikit-learn in the PR:
https://github.com/microsoft/responsible-ai-toolbox/pull/2507